src/test/java/org/minions/devfund/damianvp/battleship/OceanTest.java

```diff
@@ -0,0 +1,254 @@
+package org.minions.devfund.damianvp.battleship;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * OceanTest class.
+ */
+public class OceanTest {
+    private Ocean ocean;
+    static final int LENGTH = 20;
+    /**
+     * Method to initialize each test.
+     */
+    @Before
+    public void setOceanClassTest() {
+        ocean = new Ocean();
+    }
+
+    /**
+     * Method to test the Ocean constructor.
+     */
+    @Test
+    public void testConstructor() {
+        assertEquals(0, ocean.getHitCount());
+        assertEquals(0, ocean.getShipsSunk());
+        assertEquals(0, ocean.getShotsFired());
+        for (int i = 0; i < LENGTH; i++) {
+            for (int j = 0; j < LENGTH; j++) {
+                assertEquals("empty", ocean.getShipArray()[i][j].getShipType());
+            }
+        }
+    }
+
+    /**
+     * Method to test placeAllShipsRandomly.
+     */
+    @Test
+    public void testPlaceAllShipsRandomly() {
+//        Ocean ocean = new Ocean();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)